pkg/sql/sem/tree/pretty_test.go

```diff
@@ -47,13 +47,18 @@ func TestPrettyData(t *testing.T) {
 		t.Fatal(err)
 	}
 	cfg := tree.DefaultPrettyCfg()
+	cfg.Align = tree.PrettyNoAlign
 	t.Run("ref", func(t *testing.T) {
 		runTestPrettyData(t, "ref", cfg, matches)
 	})
-	cfg.Align = true
+	cfg.Align = tree.PrettyAlignAndDeindent
 	t.Run("align", func(t *testing.T) {
 		runTestPrettyData(t, "align", cfg, matches)
 	})
+	cfg.Align = tree.PrettyAlignOnly
+	t.Run("align", func(t *testing.T) {
+		runTestPrettyData(t, "align2", cfg, matches)
```

Comment:
`align` and `align2` in the file names are a bit vague. Could we do something descriptive instead? Maybe `deindent` and `align`? Or `align-deindent` and `align-only`? Also I don't care too much so leave as is if you like.

Reply:
Went for more descriptive names. Thanks for the suggestion.